compiler/circle-mpqsolver/src/bisection/BisectionSolver.h

```diff
@@ -75,9 +81,11 @@ class BisectionSolver final : public MPQSolver
                  const std::string &def_quant, core::LayerParams &layers);
 
 private:
-  float _qerror = 0.f; // quantization error
+  float _qerror_ratio = 0.f; // quantization error ratio
```

Comment:
IIUC, this value has to be not changed after construction. Can you make it const?

Reply:
> IIUC, this value has to be not changed after construction. Can you make it const?

ok.